ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcHttpServiceTlsMisconfigurationTest.java

```diff
@@ -121,6 +122,15 @@ public void beforeEach() {
                     Collections.emptyMap()));
   }
 
+  private Path createKnownClientsFile() throws Exception {
```

Comment:
Is this method a duplicate also added in `JsonRpcHttpServiceTlsTest.java`, util method for reuse?

Reply:
let me check. Probably a candidate to move in common testSupport or utility class.

Reply:
This has been moved to a separate utility class.